### ISSUE TITLE
Docker

### ISSUE THREAD

**Issue by matheus-rodrigues00** [whizkydee/olaolu.dev#115]
I believe it would be helpful to use Docker on this project, it has lots of unpredictable behaviors running on different environments, like macOS and node-gyp conflicts.

**Comment by Countyy** [comment]
I agree, had many problems trying to run this project without Docker. Many of those could be solved with an image.
